test_configs/tcp/tcp.exclude

```diff
@@ -0,0 +1,14 @@
+# Regex patterns of tests to exclude in runfabtests.sh
+
+cq_data
+dgram
+rdm_rma
```

Comment:
Shouldn't this test be passing?

Reply:
Currently tcp does not support FI_RMA_EVENT.

Reply:
Does rxm support FI_RMA_EVENT when the underlying provider doesn't?

Reply:
No, RxM is dependent on the underlying provider for FI_RMA_EVENT. So this test can be excluded.

Reply:
I think shmem uses RMA event.  We should check this.  If so, RxM may need to add support for it.  The underlying providers won't have it.

Reply:
> If so, RxM may need to add support for it.

Should it be an oob message from initiator to target?

Reply:
I'm not sure we have the same definition of OOB message, but there would need to be some sort of protocol message to indicate this.  Hopefully only remote writes need to be counted.  But checking with Kayla, it sounds like shmem has a fallback mechanism that doesn't use rma events for Cray.

Reply:
> I'm not sure we have the same definition of OOB message

Sorry, I meant some sort of protocol message.